flink-table/flink-table-api-java-bridge/src/main/java/org/apache/flink/connector/print/table/PrintTableSinkFactory.java

```diff
@@ -113,19 +125,35 @@ public ChangelogMode getChangelogMode(ChangelogMode requestedMode) {
         @Override
         public SinkRuntimeProvider getSinkRuntimeProvider(DynamicTableSink.Context context) {
             DataStructureConverter converter = context.createDataStructureConverter(type);
+            staticPartitions.forEach(
+                    (key, value) -> {
+                        printIdentifier = null != printIdentifier ? printIdentifier + ":" : "";
+                        printIdentifier += key + "=" + value;
+                    });
```

Comment:
These changes are not covered. Consider adding a test in `PrintConnectorITCase` so that the result shows this `printIdentifier`.

Reply:
Let me add more tests.